fixtures/log.py

```diff
@@ -41,15 +41,8 @@ def pytest_runtest_logreport(report):
         path, lineno, domaininfo = report.location
         test_status = _test_status(_format_nodeid(report.nodeid, False))
         if test_status == "failed":
-            try:
-                logger().info(
-                    "Managed providers: {}".format(
-                        ", ".join(pytest.store.current_appliance.managed_providers)))
-            except KeyError as ex:
```

Comment:
Nope, it's not because its now handled in the right place - in managed_providers (see below) which now returns an empty list if it's unable to fetch them.